### PR TITLE
GitHub actions

### PR DESCRIPTION
**Pull request by whiteshadow-gr**
Added CI workflows. Dunno if 2 workflows are needed or not

### CODE REVIEW

File: .github/workflows/dev.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Sbt clean, compile and test
+        run: |
+          export PATH=$PATH:$HOME/.local/bin
+          sbt clean
+          sbt compile
+          sbt test:compile
```
Comment (benshaw) [inline .github/workflows/dev.yml:18]: why compile tests before running them ?

File: .github/workflows/dev.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+          sbt coverage test
```
Comment (benshaw) [inline .github/workflows/dev.yml:19]: is coverage configured ? if not im not sure what doing this will do

**Comment by dataswifty** [comment]
Can we cancel this @whiteshadow-gr ?

**Comment by whiteshadow-gr** [comment]
I would say so but Probably the best person to judge will be @pperzyna now :) . I don't have any strong feelings about it.

**Pull request closed without merging**